src/dapps/preview/wearable-definition.ts

```diff
@@ -0,0 +1,11 @@
+import { RepresentationDefinition, Wearable } from '../..'
+
+/** @alpha */
+export type WearableDefinition = Omit<Wearable, 'data'> & {
```

Comment:
Is there any reason why `WearableDefinition`, `RepresentationDefinition` and `PreviewConfig` don't have schemas and validators?

Reply:
Yes, as explained it on the PR description:

> [...] it was quite complex to generate schemas for them without repeating code. I tried importing the original types and modify just the necessary parts using spreads but that screwed up the typings of the schema (ended up with huge union types that broke the new schemas). I tried copying the whole schema without using spreads, just reusing the imported schemas as they are when possible, but ended up with lots of circular dependencies. There was no way to make this work without lots of anys or @ts-ignores, which kinda defeats the purpose. Since the types in question (WearableDefinition and RepresentationDefinition) are NOT meant for deployments, I feel like having an schema is not big of a deal (I don't think anyone would be using them, since I'm adding them to type the response of the lambdas in the UI).

Reply:
I can't create an schema for it without an schema for the others

Reply:
I would not be able to type a few properties that are of type `WearableDefinition`